### DISCUSSION TITLE
Mqtt Last Will and Testament use on Project

### DISCUSSION THREAD

**Discussion by FilipeCarvalhedo** [zwave-js/zwave-js-ui#2796] [Q&A] [answered]
Hi, I'm using the zwave-js to control some devices and I need a way to know if the gateway is offline. I have researched a little bit how to do it with mqtt and I found out the Last Will and Testament. Does this project have something similar?

**Comment by robertsLando** [comment, marked as answer]
Sure, lastWill testament topic is:

```
<mqtt_prefix>/_EVENTS_/ZWAVE_GATEWAY-<mqtt_name>
```

Payload: 

```
{
  "value": false
}
```
